Watchman.Engine/Generation/Dynamo/IOrphanTablesReporter.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using Watchman.Configuration;
+
+namespace Watchman.Engine.Generation.Dynamo
+{
+    public interface IOrphanTablesReporter
```

Comment:
These interfaces are so I can write the end to end tests of dynamo alarm creation but ignore these parts for now